OrchardCore.sln

```diff
@@ -432,6 +432,8 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "OrchardCore.Themes", "Orcha
 		src\OrchardCore.Themes\Directory.Build.targets = src\OrchardCore.Themes\Directory.Build.targets
 	EndProjectSection
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "OrchardCore.Security", "src\OrchardCore.Modules\OrchardCore.Security\OrchardCore.Security.csproj", "{B02C00A7-33C2-4FEE-9D0F-B14C349ADB68}"
```

Comment:
This could come in this PR, but after we finalize everything, or we could right the docs later

Reply:
Don't do it in a separate PR because until it's also documented (and thus more accessible to users) the feature is not done. But it can be done at the end too.

Reply:
When do you intend to add some docs?

Reply:
I will try to start on it today